docs/guides/payments.rst

```diff
@@ -178,52 +164,74 @@ Example
+            'currency': response.currency,
+            'error': get_error(response),
+            'raw_response': get_payment_gateway_response(response),
+        }
+
+process_payment(payment_information, \*\*connection_params)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Used for the checkout process, it should perform all the necessary
+steps to process a payment. It should use already defined functions,
+like authorize and capture.
+
+Example
+"""""""
+
+.. code-block:: python
+
+    def process_payment(
+            payment_information: Dict,
+            **connection_params: Dict) -> Dict:
+
+        # Authorize, update the token, then capture
+        authorize_response = authorize(
+            payment_information, **connection_params)
+        payment_information['token'] = authorize_response['transaction_id']
+
+        capture_response = capture(
+            payment_information, **connection_params)
+
+        # Return a list of responses, each response must be json serializable
+        return [authorize_response, capture_response]
 
 Parameters
 ^^^^^^^^^^
 
-+-----------------------+-------------+------------------------------------------------------------------------------------+
-| name                  | type        | description                                                                        |
-+-----------------------+-------------+------------------------------------------------------------------------------------+
-| ``payment``           | ``Payment`` | Payment instance, for which the transaction will be created.                       |
-+-----------------------+-------------+------------------------------------------------------------------------------------+
-| ``client_token``      | ``str``     | Unique client's token that will be used as his indentifier in the payment process. |
-+-----------------------+-------------+------------------------------------------------------------------------------------+
-| ``connection_params`` | ``dict``    | List of parameters used for connecting to the payment's gateway.                   |
-+-----------------------+-------------+------------------------------------------------------------------------------------+
-| ``amount``            | ``Decimal`` | Amount of Money to be refunded/captured.                                           |
-+-----------------------+-------------+------------------------------------------------------------------------------------+
++-------------------------+----------+------------------------------------------------------------------------------------+
+| name                    | type     | description                                                                        |
++-------------------------+----------+------------------------------------------------------------------------------------+
+| ``payment_information`` | ``dict`` | Payment information, containing the tokens, amount, currency and billing.          |
```

Comment:
What's the key of ```payment_information```, ```tokens```, ```amount```, ```currency``` and ```billing```? What'is the type of them such as for ```billing```. Maybe it worth to mention it in the docs, so the new users to saleor can implement new gateway easier.

Reply:
Added example data that is sent (with comments) and an additional table with fields explained for the gateway response.